terraform/shared/opensearch/opensearch.tf

```diff
@@ -0,0 +1,111 @@
+resource "aws_ssm_parameter" "opensearch" {
+  name  = var.service_name
+  type  = "String"
+  value = local.connection_config
+
+  tags = merge(
+    var.tags,
+    {
+      Name = "parameter-storage"
+    }
+  )
+}
+
+resource "random_password" "es_password" {
```

Comment:
As we switched from the ElasticSearch to OpenSearch, then I think it is better to use "os" prefix in names/variables

Reply:
done